test/db/cmd/cmd_help

```diff
@@ -242,4 +242,36 @@ EOF
 EXPECT=<<EOF
 0x116e
 EOF
-RUN
\ No newline at end of file
+RUN
+
+NAME=recursive help
+FILE=-
+CMDS=<<EOF
+?* zi
+EOF
+EXPECT=<<EOF
+Usage: [0mzi[0m[j*qi][0m   # Show zignatures matching information[0m
```

Comment:
isn't `scr.color` usually disabled when running tests?

Reply:
yes; but unless that is a bug, that is the output :/

Reply:
I think it's because you called rz_cmd_get_help with true instead of passing the value of scr.color

Reply:
oh, maybe.